cpp/src/reductions/scan/scan_inclusive.cu

```diff
@@ -275,7 +275,9 @@ std::unique_ptr<column> scan_inclusive(
 {
   auto output = scan_agg_dispatch<scan_dispatcher>(input, agg, null_handling, stream, mr);
 
-  if (null_handling == null_policy::EXCLUDE) {
+  if (agg->kind == aggregation::EWMA) {
+    return output;
```

Comment:
Is EWMA the only aggregation that's doing its own null handling (because it has special logic)? Is that why this special case exists here?

Reply:
That's correct. I got the idea from how `RANK` worked at the time I started implementing this. Happy to change it to work differently if theres an obvious way that would be better, but I am a little fuzzier on this area of the pipeline in general so it's harder to tell if something is a good idea or not.

Reply:
Is null handling a requirement? If there's no precedence here, we should wait until we have a use case. It seems odd that `mean` would handle nulls when we suspect the others will not or cannot.

Reply:
I believe all of the other scan functions handle nulls, its just that a lot of the null handling is common to the scan functions (min, max, prod, sum) so it can be abstracted out. Rank and ewm have to do their own thing.